_sources/Files/AlternativeFileReadingMethods.rst

```diff
@@ -64,45 +64,49 @@ until later in this course. We don't need them for handling data from files.
 
 **Check your Understanding**
 
-.. datafile:: travel_plans.txt
-   :hide:
-
-   This summer I will be travelling.
-   I will go to...
-   Italy: Rome
-   Greece: Athens
-   England: London, Manchester
-   France: Paris, Nice, Lyon
-   Spain: Madrid, Barcelona, Granada
-   Austria: Vienna
-   I will probably not even want to come back! 
-   However, I wonder how I will get by with all the different languages.
-   I only know English!
-
-.. datafile:: school_prompt.txt
-   :hide:
-
-   Writing essays for school can be difficult but
-   many students find that by researching their topic that they
-   have more to say and are better informed. Here are the university
-   we require many undergraduate students to take a first year writing requirement
-   so that they can
-   have a solid foundation for their writing skills. This comes
-   in handy for many students.
-   Different schools have different requirements, but everyone uses
-   writing at some point in their academic career, be it essays, research papers,
-   technical write ups, or scripts.
-
-.. datafile:: emotion_words.txt
-   :hide: 
-
-   Sad upset blue down melancholy somber bitter troubled
-   Angry mad enraged irate irritable wrathful outraged infuriated
-   Happy cheerful content elated joyous delighted lively glad
-   Confused disoriented puzzled perplexed dazed befuddled
-   Excited eager thrilled delighted
-   Scared afraid fearful panicked terrified petrified startled
-   Nervous anxious jittery jumpy tense uneasy apprehensive
+.. raw:: html
```

Comment:
I think these would be more convenient if we moved them in between the exercises so they were right above.  Or now that you are using the raw trick we could embed them right in the questions.

Its interesting that datafiles are actually one thing where we wouldn't mind duplicate ids as long as the contents are in sync...  I'll have to think about that a bit more.